Makefile

```diff
@@ -102,16 +111,26 @@ build-app: require-APP build-app-$(APP)
 build-app-%: nix
 	nix-build nix/default.nix -A $(subst build-app-,,$@) -o result-$(subst build-app-,,$@) --fallback
 
+copy-app-%:
+	$(eval APP_TMP := $(shell mktemp -d --tmpdir=$$PWD/tmp $(APP).XXXXX))
+	cp -rf result-$(APP)/* $(APP_TMP)
 
+configure-staging-%: copy-app-$(APP)
+	@for v in $(APP_STAGING_ENV_$(APP)) ; do \
+		sed -i -e "s,$${v/\=/,},g" $(APP_TMP)/bundle.js ; \
+	done
+
+configure-production-%: copy-app-$(APP)
+	@for v in $(APP_PRODUCTION_ENV_$(APP)) ; do \
+		sed -i -e "s,$${v/\=/,},g" $(APP_TMP)/bundle.js ; \
+	done
```

Comment:
wouldn't it be better to sed ``index.html`` adding eg. data-backend-dashboard-url="http:/..." and explicitly read it from ``index.js``